scripts/set-performance

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Is there a source of why we are updating these settings that we may ref in the comment, if not for at least my own edification?